pkg/ffcapi/block_info_by_number.go

```diff
@@ -22,6 +22,7 @@ import (
 
 type BlockInfoByNumberRequest struct {
 	BlockNumber        *fftypes.FFBigInt `json:"blockNumber"`
+	AllowCache         bool              `json:"allowCache"`
```

Comment:
Requires trivial change to `firefly-evmconnect` to propagate this to an existing internal boolean